articles/api/management/v2/tokens/manually.md

```diff
@@ -0,0 +1,101 @@
+---
+description: How to manually generate and use a token for the Management API
+section: apis
+crews: crew-2
+toc: true
+---
+# Manually generate and use a token for the Management API
+
+In this article, we will cover how you can generate the token required to call the Management API manually. We will then cover how you can use your newly-obtained token.
+
+## Token validity
+
+By default, the Management API Token is valid for 24 hours. Once the token expires, you will need to obtain a new token.
+
+If this doesn't work for you, you can either:
+
+1. [Change the validity period of the token](#2-get-the-token)
+2. [Automate the token request process](/api/management/v2/tokens/programatically).
+
+## Generate a token for the Management API
+
+To generate a token for use with the Management API, you will need to:
+
+1. Create and authorize a client
+2. Get the token by copying it for later use
+
+## 1. Create and authorize a client
```

Comment:
For the manual process you don't have to do this I think. A client (named `API Explorer Client`) is automatically created once the `Auth0 Management API` is created at Dashboard > API (which is also automatic I think)

Reply:
You can just leave two sections in the doc: how to get a token, how to use a token

Reply:
@khorne3 Can you cross check with @cabralmartin ?

Reply:
Confirmed behavior w/ Crew 2